Scrape_Twitter_R.Rmd

```diff
@@ -0,0 +1,203 @@
+# Scrape Twitter data using R
+
+Senqi Zhang
+
+```{r, message=FALSE}
+# install.packages("twitteR")
+# install.packages("rtweet")
+library(dplyr)
+library(twitteR) #Note there is only one big capital R at the end
+library(rtweet)
+```
+
+Social media is an emerging field of study for researchers. During the pandemic, lots of studies carried out analysis using Twitter data. In this tutorial, I want to introduce the basics of scraping Twitter data using R. I will introduce two libraries `rtweet` and `twitteR`. Note that these two libraries do not complement each other. `rtweet` actually contains all functionality in `twitteR` and more. It is updated and maintained more recently, and I would always suggest using `rtweet` for collecting Twitter data. However, for this tutorial, I will still introduce `twitteR`.
+
+## 1.Preparation
```

Comment:
Please remove the numbering in subheadings. They'll be automatically generated when rendering.

Reply:
These numberings also need to be removed.